config/routes.rb

```diff
@@ -17,16 +17,7 @@
         get "unsubscribe_all", on: :collection
       end
       namespace :subscriptions do
-        resource :bulk, controller: :bulk, only: [:update, :destroy]
-      end
-    end
-  end
-
-  namespace :api, defaults: {format: :json} do
-    scope "/v:api_version" do
-      resources :hackathons, only: [:index, :show]
-      scope module: :hackathon do
-        resources :subscriptions, only: :create
+        resource :bulk, only: [:update, :destroy]
```

Comment:
Does this work without `controller: :bulk`? I believe it would try to find a `bulks` controller, which doesn't exist

Reply:
Ah, you're right